packages/dso-toolkit/components/Componenten/badge/badge.njk

```diff
@@ -1 +1,5 @@
 <span {{ className('dso-badge', [modifier, 'badge-' + modifier]) }}>{% if status -%}<span class="sr-only">{{ status }}: </span>{% endif -%}{{ label }}</span>
+
+<dso-badge level="{{ modifier }}">
+  {% if status -%}<span class="sr-only">{{ status }}: </span>{% endif -%}{{ label }}
```

Comment:
`status` moet een **optionele** `@Prop()` op het Badge component worden. In het component komt logica waarmee voorwaardelijk deze `.sr-only` wordt getoond.

Dit is het eerste component wat gebruik maakt van `.sr-only`, deze styling leeft nog niet in `@dso-toolkit/styling`. De class `.sr-only` mag uit `packages\dso-toolkit\libs\bootstrap\_scaffolding.scss` verplaatst worden naar `@dso-toolkit/styling/scaffolding/sr-only` met een import in `@dso-toolkit/styling/scaffolding.scss`. Omdat dit component al `@import "@dso-toolkit/styling/scaffolding";` doet, kan je vanaf nu gebruik maken van `.sr-only`.